cluster/gce/config-test.sh

```diff
@@ -156,9 +156,6 @@ if [[ (( "${KUBE_FEATURE_GATES:-}" = *"AllAlpha=true"* ) || ( "${KUBE_FEATURE_GA
   RUN_CONTROLLERS=${RUN_CONTROLLERS:-*,endpointslice}
 fi
 
-# Optional: set feature gates
-FEATURE_GATES=${KUBE_FEATURE_GATES:-}
```

Comment:
same as above!

Reply:
Please refer to the explanation on why this had to be removed [here](https://github.com/kubernetes/kubernetes/pull/112980#discussion_r1007828904).